db/migrate/20160920144053_change_profile_to_json.rb

```diff
@@ -0,0 +1,45 @@
+class ChangeProfileToJson < ActiveRecord::Migration
+  class Profile < ActiveRecord::Base
+  end
+
+  def up
+    add_column :profiles, :encrypted_pii, :text
```

Comment:
Why `text` and not Postgres native `jsonb` column type? http://nandovieira.com/using-postgresql-and-jsonb-with-ruby-on-rails